packages/devtools_app/lib/src/memory/memory_screen.dart

```diff
@@ -969,9 +975,9 @@ class MemoryBodyState extends State<MemoryBody>
 
   List<Widget> displayVmDataInHover(ChartsValues chartsValues) {
     const rssDisplay = 'RSS';
-    const capacityDisplay = 'Capacity';
-    const usedDisplay = 'Used';
-    const externalDisplay = 'External';
+    const capacityDisplay = 'Allocated';
+    const usedDisplay = 'Dart/Flutter';
+    const externalDisplay = 'Flutter engine/rendering';
```

Comment:
This now seems too long.  I wonder if a more accurate name might be "Dart/Flutter Native" as this more accurate.  It contains native objects that a Dart embedder would manage with a Dart wrapper .

Reply:
Alex can you change this label "Flutter engine/rendering" to "Dart/Flutter Native"? Plus update the screen shots?

I think after this the PR will be fine.  I can commit and follow up with new goldens so the bots turn green.

Reply:
Yes, I changed those now. For me Dart/Flutter Native seems too close to Dart/Flutter, so it might get confusing for some developers. Yet I also agree that Flutter engine/rendering is too long string.